operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -76,14 +87,20 @@ func getPersistenceByTarget(central *platform.Central, target PVCTarget) *platfo
 	switch target {
 	case PVCTargetCentral:
 		return nil
-	case PVCTargetCentralDB:
+	case PVCTargetCentralDB, PVCTargetCentralDBBackup:
 		if !central.Spec.Central.ShouldManageDB() {
 			return nil
 		}
 		dbPersistence := central.Spec.Central.GetDB().GetPersistence()
 		if dbPersistence == nil {
 			dbPersistence = &platform.DBPersistence{}
 		}
+
+		if target == PVCTargetCentralDBBackup {
+			claimName := dbPersistence.PersistentVolumeClaim.ClaimName
+			backupName := fmt.Sprintf("%s-backup", *claimName)
+			dbPersistence.PersistentVolumeClaim.ClaimName = &backupName
```

Comment:
I don't think I understand the intent of this code...

Reply:
The PVC configuration for the backup volume is the same as for the main data volume, due to using `GetPersistence` for both. But the claim name has to be different of course, hence this modification.

Reply:
I think I got it a bit better now.